blog/_posts/2023-01-31-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,259 @@
+
+### Language, Compiler, Standard Library
+
+**We overhauled and restarted the [Scala Improvement Process](https://docs.scala-lang.org/sips/).** Since last
+summer, the SIP Committee members meet monthly to decide the evolution of the language. As we explained
+in the [announcement]({% link blog/_posts/2022-07-13-scala-improvement-process-reloaded.md %}), the main changes of
+the new process are the following. First, the proposals are thoroughly discussed by a team of three reviewers, publicly
+on [GitHub](https://github.com/scala/improvement-proposals) before the whole Committee votes on them during their
+monthly meetings. Second, there is a new stage where proposals are accepted as experimental features before the Committee
+votes again to promote them to stable features. The new Scala Improvement Process is the first result of our governance
+strategy whose goal is to establish all the necessary decision-making processes for the community, by the community.
+
+**We improved meta-programming, especially [generic Mirror](https://docs.scala-lang.org/scala3/reference/contextual/derivation.html#mirror)
+synthesis.** Mirrors are critical to metaprogramming in Scala 3, they enable implementation of type class derivation without
+advanced metaprogramming techniques such as macros. Mirror synthesis now supports more cases
+([generic tuples](https://github.com/lampepfl/dotty/pull/15250),
+[local and inner classes](https://github.com/lampepfl/dotty/pull/15847)), is more reliable
+([dotty#15279](https://github.com/lampepfl/dotty/pull/15279),
+[dotty#15404](https://github.com/lampepfl/dotty/pull/15404),
+[dotty#15814](https://github.com/lampepfl/dotty/pull/15814)), and provides
+[more detailed explanations in case of failure](https://github.com/lampepfl/dotty/pull/15164).
+
+**We generalized the definition of methods in Scala** by allowing type parameters to be interleaved with regular parameters.
+This change notably allows methods to take type parameters whose bounds depend on value parameters. You can learn more
+about the motivation behind this change in the proposal document:
+[SIP-47 - Clause Interleaving](https://docs.scala-lang.org/sips/clause-interleaving.html). Last, we lifted a current
+language restriction that prevents eta-expansion to be applied to polymorphic methods. You can learn more about this
+change in the proposal document:
+[SIP-49 - Polymorphic Eta-Expansion](https://docs.scala-lang.org/sips/polymorphic-eta-expansion.html). Both proposals
+have been accepted by the SIP Committee and are under development in the compiler.
+
+### Documentation and MOOCs
+
+**We improved the usability of the Scala website (this website).** The website had not seen a major rework since 2013,
+and it does not represent the full picture that we would like Scala newcomers to see. To this effect, we have removed
+the split of Scala 2 vs Scala 3 on [the landing page](https://github.com/scala/scala-lang/pull/1344) and on
```

Comment:
```suggestion
the split of Scala 2 vs Scala 3 on [the landing page](https://github.com/scala/scala-lang/pull/1344) and in
```

We should either say "in the documentation" or "on the documentation page", but I think you want the former.

Reply:
Done in 79c0d585